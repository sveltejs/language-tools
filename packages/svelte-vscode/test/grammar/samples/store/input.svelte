<script>
import './$types';
import '$lib/foo';
$bar;
</script>

{$bar}
