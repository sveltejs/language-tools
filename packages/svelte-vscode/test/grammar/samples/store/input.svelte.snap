><script>
#^ source.svelte meta.script.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^^^^ source.svelte meta.script.svelte meta.tag.start.svelte entity.name.tag.svelte
#       ^ source.svelte meta.script.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
>import './$types';
#^^^^^^^^^^^^^^^^^^^ source.svelte meta.script.svelte meta.embedded.block.svelte source.js
>import '$lib/foo';
#^^^^^^^^^^^^^^^^^^^ source.svelte meta.script.svelte meta.embedded.block.svelte source.js
>$bar;
#^ source.svelte meta.script.svelte meta.embedded.block.svelte source.js punctuation.definition.variable.svelte
# ^^^^^ source.svelte meta.script.svelte meta.embedded.block.svelte source.js
></script>
#^^ source.svelte meta.script.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#  ^^^^^^ source.svelte meta.script.svelte meta.tag.end.svelte entity.name.tag.svelte
#        ^ source.svelte meta.script.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>
>{$bar}
#^ source.svelte punctuation.section.embedded.begin.svelte
# ^ source.svelte meta.embedded.expression.svelte source.ts punctuation.definition.variable.svelte
#  ^^^ source.svelte meta.embedded.expression.svelte source.ts
#     ^ source.svelte punctuation.section.embedded.end.svelte
>